app.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+
+class App
+  attr_accessor :books, :people, :rentals
+  attr_reader :main
+
+  def initialize(books = [], people = [], rentals = [])
+    @books = books
+    @people = people
+    @rentals = rentals
+  end
+
+  def book_list
+    puts 'Books list'
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def people_list
+    puts 'People list'
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp.to_i
+    case person_type
+    when '1' then create_student
+    when '2' then create_teacher
```

Comment:
- [ ] I'm having an error occur when I try to insert the numbers '1' or '2', I think it's because you're introducing them to be inserted as strings, and not integers. Try to remove the double quotes.

Reply:
done